app/src/molecules/MoleculeBreadcrumb.re

```diff
@@ -0,0 +1,23 @@
+open Atom;
+open DesignSystem;
+open Particle;
+open Router;
```

Comment:
It depending on Router might indicate it was extracted out to early in a molecule.

Reply:
We have the same problem in the `Link` atom. I'd rather put it in my backlog and work on it in a separate PR, as this one has already gotten quite big.

Reply:
Safe.